contracts/deploy/plume/001_woeth.js

```diff
@@ -0,0 +1,84 @@
+const { deployOnPlume } = require("../../utils/deploy-l2");
+const {
+  deployWithConfirmation,
+  withConfirmation,
+} = require("../../utils/deploy");
+const { getTxOpts } = require("../../utils/tx");
+const addresses = require("../../utils/addresses");
+
+module.exports = deployOnPlume(
+  {
+    deployName: "001_woeth",
+  },
+  async ({ ethers }) => {
+    const { deployerAddr } = await getNamedAccounts();
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+
+    // Deploy Proxy
+    await deployWithConfirmation("BridgedWOETHProxy", []);
+    const cWOETHProxy = await ethers.getContract("BridgedWOETHProxy");
+    console.log("BridgedWOETHProxy address:", cWOETHProxy.address);
+
+    // Deploy Bridged WOETH Token implementation
+    await deployWithConfirmation("BridgedWOETH", []);
+    const cWOETHImpl = await ethers.getContract("BridgedWOETH");
+    console.log("BridgedWOETH address:", cWOETHImpl.address);
+
+    const cWOETH = await ethers.getContractAt(
+      "BridgedWOETH",
+      cWOETHProxy.address
+    );
+
+    const initData = cWOETHImpl.interface.encodeFunctionData(
+      "initialize()",
+      []
+    );
+
+    // Initialize the proxy
+    // prettier-ignore
+    await withConfirmation(
+      cWOETHProxy
+        .connect(sDeployer)["initialize(address,address,bytes)"](
+          cWOETHImpl.address,
+          deployerAddr, // Pretend Deployer is Governor for now
```

Comment:
no multisig provider on Plume is quite annoying

Reply:
Actually, we have multisig and timelock. The governance is transferred for this token to the multisig. I had to manually do it, since I deployed the multisig a bit later